src/Components/Login/Login.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {screen, fireEvent, render} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Login from './Login';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('Login Component', () => {
+  it('should have two input fields and a button', () => {
+  render (<Login />);
+
+  const emailInput = screen.getByPlaceholderText('Username');
+  const passwordInput = screen.getByPlaceholderText('Password');
+  const submitButton = screen.getByRole('button', {name: 'Submit'});
+
+  expect(emailInput).toBeInTheDocument();
+  expect(passwordInput).toBeInTheDocument();
+  expect(submitButton).toBeInTheDocument();
+})
+
+  it('should check the input is clear after submit', () => {
+    render(<BrowserRouter><Login/></BrowserRouter>);
```

Comment:
Have y'all started implementing Router? It doesn't really matter for unit tests, but be aware that BrowserRouter isn't always the best option for a wrapper while testing. We'll talk more about it tomorrow.